CHANGES.md

```diff
@@ -1,3 +1,7 @@
+#### 2.5.5
```

Comment:
My only comment is: I'm about to publish 2.5.5, I think you should go for 2.6.0 as you're pushing a contract change

Reply:
Good catch!